### ISSUE TITLE
Inconsistent ValueError related to T3AFG instrument check_set_errors

### ISSUE THREAD

**Issue by scandey** [pymeasure/pymeasure#833]
It seems like every so often, check_set_errors leads to a portion of the outgoing command being fed back into the error checker. 

More details forthcoming tomorrow, I just wanted to track this here in case it turns out to be a code issue rather than an instrument implementation issue. 

```
  File "<input>", line 1, in <module>
  File "/Users/scottcandey/Documents/TRACERS/tracers_pygse/__pypackages__/3.10/lib/pymeasure/instruments/common_base.py", line 204, in __setattr__
    super().__setattr__(name, value)
  File "/Users/scottcandey/Documents/TRACERS/tracers_pygse/__pypackages__/3.10/lib/pymeasure/instruments/common_base.py", line 88, in __set__
    self.fset(obj, value, **kwargs)
  File "/Users/scottcandey/Documents/TRACERS/tracers_pygse/__pypackages__/3.10/lib/pymeasure/instruments/common_base.py", line 470, in fset
    self.check_errors()
  File "/Users/scottcandey/Documents/TRACERS/tracers_pygse/__pypackages__/3.10/lib/pymeasure/instruments/channel.py", line 110, in check_errors
    return self.parent.check_errors()
  File "/Users/scottcandey/Documents/TRACERS/tracers_pygse/__pypackages__/3.10/lib/pymeasure/instruments/instrument.py", line 207, in check_errors
    if int(err[0]) != 0:
ValueError: invalid literal for int() with base 10: 'C1:BSWV WVTP'
```

**Comment by scandey** [comment]
I realized this is likely due to an automated process running in the same software that regularly requests the status of the function generator approximately every half second. 

If I manually set a control that happens to collide in time with the status check, the error checking function receives the result of the status check query rather than the error query. That probably also explains some of the occasional timeout errors I was seeing.

Probably running into awkward thread issues in my application, rather than anything specific to pymeasure.
